src/frontend/apps/lti_site/components/RedirectOnLoad/RedirectVideo.spec.tsx

```diff
@@ -123,13 +128,61 @@ describe('RedirectVideo', () => {
             render: () => <span>video player</span>,
           },
           { path: VIDEO_WIZARD_ROUTE(), render: () => <span>wizard</span> },
+          {
+            path: VIDEO_WIZARD_ROUTE(VideoWizzardSubPage.createVideo),
+            render: () => <span> VOD creation</span>,
+          },
         ],
       },
     });
 
     screen.getByText('wizard');
   });
 
+  it('redirects to the wizard VOD creation when the user has update permission and video title is set', () => {
```

Comment:
Are update permission and set title really a condition to be redirected to the wizard ?
Anyway, uploadState.INITIALIZING is missing here.
```suggestion
  it('redirects to the wizard VOD creation when the video update state is initializing, () => {
```

Reply:
No the title is wrong, I made this in an old commit where I made the redirection based on the video title and that was wrong.